lib/build/builder.js

```diff
@@ -239,14 +233,14 @@ class Builder {
           // we'll take care of this one, skip over the -o flag
           i++
           continue
-        } else if (parsed[i] === '-c') {
+        } else if (parsed[i] === '-c' || parsed[i] === '-i') {
```

Comment:
Which version of Go added the `-i` flag to `go test`?  Is it supported in 1.10?

Reply:
[1.9](https://github.com/golang/go/blob/release-branch.go1.9/src/cmd/go/internal/test/test.go#L104), [1.10](https://github.com/golang/go/blob/release-branch.go1.10/src/cmd/go/internal/test/test.go#L157) and obviously [1.11](https://github.com/golang/go/blob/release-branch.go1.11/src/cmd/go/internal/test/test.go#L154).

Interestingly this even dates back to [1.0](https://github.com/golang/go/blob/release-branch.go1/src/cmd/go/test.go#L62)!